Samples/PowerShell/functions/func_ModernDashboards.ps1

```diff
@@ -32,69 +160,80 @@
     Validated: Orion Platform 2020.2.5
 
     TBD List:
-        * -PassThru [switch]
+        * -JsonOnly [switch] (Completed)
```

Comment:
Consider removing completed TBD items instead of marking them as completed.

Reply:
Removed - and updated the Last Updated and Platform Version to what I'm actually using.